splinterd/packaging/splinterd.toml.example

```diff
@@ -13,44 +13,133 @@
 # limitations under the License.
 
 # Config file format version
-version = "1"
+#version = "1"
+
+# Specifies the directory containing Splinter configuration files.
+#config_dir = "/etc/splinter"
+
+# Specifies the storage directory for Splinter state. This option overrides the
+# SPLINTER_STATE_DIR environment variable, if set.
+#state_dir = "/var/lib/splinter"
+
+# Specifies the database used for Splinter state.
+#database = "splinter_state.db"
 
 # Friendly identifier for this node. Must be unique on the network
-node_id = "acme-node-000"
+#node_id = ""
```

Comment:
Should we add a note that this must match the value in the file in /var/lib/splinter?